.github/workflows/codeql-analysis.yml

```diff
@@ -27,7 +26,7 @@ jobs:
           submodules: true
 
       - name: Initialize CodeQL
-        uses: github/codeql-action/init@v1
+        uses: github/codeql-action/init@v2
```

Comment:
>uses: github/codeql-action/init@v2 [](http://example.com/codeflow?start=8&length=34)

Switching to the latest version of CodeQL, right?

Reply:
yes, dependabot created it